### ISSUE TITLE
Importing error.

### ISSUE THREAD

**Issue by guyedry1** [DemonMartin/tlsClient#6]
Hi. Thanks a lot for this amazing library!
Im using pkg and it requires to use ESM module.
im import using: 
const tlsClient =require('tlsclientwrapper'); and getting this error: 
Error [ERR_REQUIRE_ESM]: require() of ES Module node_modules\tlsclientwrapper\index.js 
  code: 'ERR_REQUIRE_ESM'
}

thanks!

**Comment by DemonMartin** [comment]
Correct. The module does not support CJS. ESM is the new standard. In the past, I tried to add CJS support to the module, but I haven’t found a way to correctly convert all the code. Therefore, I decided to keep ESM.

I have tried multiple ways to create an .exe file, and none have worked.

Deno Compiler, Bun Compiler, NodeJS Compiler, Nexe, PKG, etc.

Feel free to open a pull request if you want to add correctly working CJS using something similar to esbuild, babel, or webpack. However, I haven’t been able to do it.

**Comment by guyedry1** [comment]
Thanks for the quick answer. I will update if i find any way.
